LearningMissionLab/Profile.cs

```diff
@@ -12,7 +12,7 @@ public class Profile : IReport
         readonly Guid _accountId;
         string _firstName;
         string _lastName;  
-        string _financialId;
+        string _financialAid;
```

Comment:
Profile should have only ONE property for ALL financial records of the person. That's why you need the FinancialInfo class. You should replace all current financial fields with FinancialInfo. Therefore, you have to remove: _financialAid, _stipend amd _salary and add FinancialInfo instead. In that sense it is pretty similar to ContactInfo.